From 9621c8a3b41466151488910ec99d8be0deb1649e Mon Sep 17 00:00:00 2001
From: Francois Daoust <fd@tidoust.net>
Date: Tue, 10 May 2022 10:36:24 +0200
Subject: [PATCH] Drop default attribute values

Default attribute values are not allowed in Web IDL, see bug report at:
https://github.com/KhronosGroup/WebGL/issues/3407
---
 ed/idl/webgl1.idl | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/ed/idl/webgl1.idl b/ed/idl/webgl1.idl
index 979218d76..351218440 100644
--- a/ed/idl/webgl1.idl
+++ b/ed/idl/webgl1.idl
@@ -516,8 +516,8 @@ interface mixin WebGLRenderingContextBase
     readonly attribute (HTMLCanvasElement or OffscreenCanvas) canvas;
     readonly attribute GLsizei drawingBufferWidth;
     readonly attribute GLsizei drawingBufferHeight;
-    attribute PredefinedColorSpace drawingBufferColorSpace = "srgb";
-    attribute PredefinedColorSpace unpackColorSpace = "srgb";
+    attribute PredefinedColorSpace drawingBufferColorSpace;
+    attribute PredefinedColorSpace unpackColorSpace;
 
     [WebGLHandlesContextLoss] WebGLContextAttributes? getContextAttributes();
     [WebGLHandlesContextLoss] boolean isContextLost();
-- 
2.36.0.windows.1

